### PR TITLE
add Timer C3 and Timer S3

### PR DESCRIPTION
**Pull request by L1cardo**
1. add Timer C3 and Timer S3.
2. add wikiUrl.
3. bump version to 1.0.4

### CODE REVIEW

File: devices/esp32c3.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "category": "ESP32",
+    "name": "Timer C3",
+    "targets": [
+      {
+        "flashingMethod": "UART",
+        "name": "esp32c3"
```
Comment (qdrk) [inline devices/esp32c3.json]: Not adding support for s3 like the firmware?

**Review by qdrk — APPROVED** [review]

**Comment by qdrk** [comment]
Could you also bump the version to 1.0.4? https://github.com/qdrk/fpvsim-timer-configurator/commit/9c5ede39ea26570ff6ad09882ab382a3822c37c7

File: devices/esp32c3.json (hunk 1 above)
Comment (L1cardo) [reply devices/esp32c3.json]: added

File: devices/licardofpv.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "category": "LicardoFPV",
+    "name": "Timer C3",
+    "targets": [
+      {
+        "flashingMethod": "UART",
+        "name": "esp32c3"
+      }
+    ],
+    "userDefines": [],
+    "wikiUrl": "https://space.bilibili.com/7403418",
```
Comment (qdrk) [inline devices/licardofpv.json]: You got a real documentation page for this? A video should also work, prefer not to direct to a channel directly.

Comment (L1cardo) [reply devices/licardofpv.json]: removed

**Review by qdrk — APPROVED** [review]

**Pull request merged**